README.md

```diff
@@ -1,29 +1,80 @@
-## An open professional network.
+# LinkedUp - An open professional network.
 
 [![Build Status](https://travis-ci.org/dfinity-lab/linkedup.svg?branch=master)](https://travis-ci.org/dfinity-lab/linkedup?branch=master)
 
 [![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/dfinity-lab/linkedup) 
 
-### Demo
+The LinkedUp sample application provides a simple implementation of an open professional network that demonstrates how to use **inter-canister calls** within a project.
 
-Install the required Node modules (only needed the first time).
+In the LinkedUp sample application, there are two canisters:
 
-```bash
-npm install
-```
+* The `linkedup` canister creates and stores basic profile information for a user, including work experience and educational background.
+* The `connectd` canister creates and stores a user's connections.
 
-Start the replica, then build and install the canisters.
+## Before you begin
 
-```bash
-dfx start --background
-dfx build
-dfx canister install --all
-```
+Before building the sample application, verify the following:
+
+* You have downloaded and installed the DFINITY Canister SDK as described in [Download and install](https://sdk.dfinity.org/docs/quickstart/quickstart.html#download-and-install).
+* You have stopped any Internet Computer network processes running on the local computer.
+
+## Demo
+
+1. Clone the `linkedup` repository.
+
+1. Change to the local `linkedup` working directory.
+
+    ```bash
+    cd linkedup
+    ```
+
+1. Install the required node modules (only needed the first time).
+
+    ```bash
+    npm install
+    ```
+
+1. Open the `dfx.json` file in a text editor and verify the `dfx` setting has same the version number as the `dfx` executable you have installed. 
+
+1. Start the replica.
+
+    ```bash
+    dfx start --background
+    ```
+
+1. Register unique canister identifiers for the `linkedup` project by running the following command:
+
+    ```bash
+    dfx canister create --all
+    ```
+
+1. Build the application by running the following command:
+
+    ```bash
+    dfx build
+    ```
+
+1. Deploy the application on the local network by running the following command:
+
+    ```bash
+    dfx canister install --all
+    ```
+
+1. Copy the canister identifier for the `linkedup_assets` canister.
+
+1. Open the `linkedup_assets` canister frontend in your web browser.
+
+    For example, if using the default localhost address and port number, the URL looks similar to this:
+
+    ```bash
+    http://localhost:8000/?canisterId=ic:03000000000000000000000000000000000179
```

Comment:
the canister ID won't look like this anymore

Reply:
updated ID